### PR TITLE
Remove the remaining .tmp files

### PR DESCRIPTION
**Pull request by hfreeb**

### CODE REVIEW

File: COMP11212/commands.tmp [hunk 1]
```diff
@@ -1,2 +1 @@
-pdflatex -shell-escape "\input{kindle.tex}"
 pdflatex -shell-escape "\input{notes.tex}"
```
Comment (Todd-Davies) [inline COMP11212/commands.tmp]: Shouldn't you delete this whole file?

File: COMP11212/preCommands.tmp [hunk 2]
```diff
@@ -1,32 +0,0 @@
-cd automata && pdflatex 10.tex && cd ..
```
Comment (Todd-Davies) [inline COMP11212/preCommands.tmp:1]: Shouldn't you delete this whole file too? (i.e. git rm <file>)

File: .gitignore~ [hunk 3]
```diff
@@ -1,29 +0,0 @@
...
-*.lot
-*.maf
-*.mtc
-*.mtc1
-*.out
-*.synctex.gz
-*.tex.bak
-*.pdf_original
-*.new
-*.class
```
Comment (Todd-Davies) [inline .gitignore~:29]: Why are you removing this? It seems pretty useful to me?

File: COMP11212/commands.tmp (hunk 1 above)
Comment (hfreeb) [reply COMP11212/commands.tmp]: I must have rebuilt it after deleting it.

File: .gitignore~ (hunk 3 above)
Comment (hfreeb) [reply .gitignore~:29]: I only deleted the ".gitignore~" that your editor probably created as a temporary file, I didn't delete the actual gitignore.

Comment (Todd-Davies) [reply .gitignore~:29]: SGTM, I think we should also add *.tmp and *~ to the .gitignore file, would you like to do it, or I can later?

Comment (Todd-Davies) [reply .gitignore~:29]: Update, *.tmp is already in there, so it's just *~ that we should probably add.

Comment (hfreeb) [reply .gitignore~:29]: Done

**Review by Todd-Davies — APPROVED** [review]

**Pull request merged**